self-hosted/upgrades/about-upgrades.md

```diff
@@ -25,6 +25,13 @@ within the Docker container. For more information, and instructions, see the
 
 <PlanUpgrade />
 
+<Highlight type="note">
+If you use the Timescale Toolkit, ensure the `timescale_toolkit` extension is on
+version 1.6.0, then upgrade the `timescaledb` extension. If required, you
+can then later upgrade the `timescale_toolkit` extension to the most
```

Comment:
Not sure if we say Timescaledb toolkit or Timescale toolkit but the extension name has the *db* suffix.

```suggestion
If you use the Timescale Toolkit, ensure the `timescaledb_toolkit` extension is on
version 1.6.0, then upgrade the `timescaledb` extension. If required, you
can then later upgrade the `timescaledb_toolkit` extension to the most
```